apps/src/sites/studio/pages/foorm/editor/index.js

```diff
@@ -22,7 +22,6 @@ $(document).ready(function() {
     <Provider store={store}>
       <FoormEditorManager
         updateFormQuestions={updateFormQuestions}
```

Comment:
This is removed as a prop?

Reply:
`updateFormData` was removed as a prop since it was only doing a dispatch to the store and we can do that within FoormEditorManager instead of passing in as a prop (and some extra functionality was added to that function in FoormEditorManager)

Reply:
Sorry, wasn't clear -- I meant `updateFormQuestions` is still being passed as a prop here to `FoormEditorManager`, but `FoormEditorManager` no longer accepts `updateFormQuestions` as a prop?

Reply:
oh whoops I'll fix that